arrows/ffmpeg/tests/test_video_input_ffmpeg.cxx

```diff
@@ -441,81 +443,82 @@ TEST_F(ffmpeg_video_input, metadata_map)
   // at a minimum this is just the video name and timestamp
   EXPECT_EQ( md_map.size(), input.num_frames() );
 
-  for ( auto md : md_map )
+  for( auto md : md_map )
   {
     std::cout << "-----------------------------------\n" << std::endl;
-    kwiver::vital::print_metadata( std::cout, *md.second[0] );
+    kwiver::vital::print_metadata( std::cout, *md.second[ 0 ] );
   }
 
-  if ( md_map.size() != input.num_frames() )
+  if( md_map.size() != input.num_frames() )
   {
     std::cout << "Found metadata on these frames: ";
-    for (auto md : md_map)
+    for( auto md : md_map )
     {
       std::cout << md.first << ", ";
     }
+
     std::cout << std::endl;
   }
 }
 
 // ----------------------------------------------------------------------------
-TEST_F(ffmpeg_video_input, no_sync_metadata)
+TEST_F ( ffmpeg_video_input, no_sync_metadata )
 {
-  std::map<int, std::set<uint64_t>> expected_md = {
-    {0, {1221515219356000, 1221515219396000, 1221515219426000}},
-    {1, {1221515219456000}},
-    {2, {1221515219486000}},
-    {3, {1221515219516000}},
-    {4, {1221515219556000}}
-  };
+  std::map< int,
+            std::set< uint64_t > > expected_md =
+  { { 0, { 1221515219356000, 1221515219396000, 1221515219426000 } },
+    { 1, { 1221515219456000 } }, { 2, { 1221515219486000 } },
+    { 3, { 1221515219516000 } }, { 4, { 1221515219556000 } } };
```

Comment:
Something very strange happened here... (And again, below.)

Reply:
```
std::map< int, std::set< uint64_t > > expected_md =
  { 
    { 0, { 1221515219356000, 1221515219396000, 1221515219426000 } },
    { 1, { 1221515219456000 } }, { 2, { 1221515219486000 } },
    { 3, { 1221515219516000 } }, { 4, { 1221515219556000 } } 
  };
```

Is this better?

Reply:
That would be... okay (certainly a huge improvement!)... However, even better would be:
```c++
std::map< int, std::set< uint64_t > > expected_md =
  { 
    { 0, { 1221515219356000, 1221515219396000, 1221515219426000 } },
    { 1, { 1221515219456000 } },
    { 2, { 1221515219486000 } },
    { 3, { 1221515219516000 } },
    { 4, { 1221515219556000 } } 
  };
```

Reply:
Ok I did that.